internal/reconcilers/certified_image_stream.go

```diff
@@ -22,20 +26,29 @@ const (
 
 type CertifiedImageStreamReconciler struct {
 	client.Client
-	Log    logr.Logger
-	Scheme *runtime.Scheme
+	Log         logr.Logger
+	Scheme      *runtime.Scheme
+	PyxisClient *pyxis.PyxisClient
```

Comment:
Good catch, I'll remove the export. I don't want to update the signature, since this updates the interface, meaning all reconcilers would have to be updated, even though not all need this.